RubberduckTests/Mocks/MockProjectBuilder.cs

```diff
@@ -320,7 +320,11 @@ private Mock<ICodeModule> CreateCodeModuleMock(string content, string name)
                 lines.TakeWhile(line => line.Contains(Tokens.Declare + ' ') || !ModuleBodyTokens.Any(line.Contains)).Count());
 
             codeModule.Setup(m => m.Content()).Returns(() => string.Join(Environment.NewLine, lines));
-            
+
+            codeModule.Setup(m => m.SimpleContentHash()).Returns(() => string.IsNullOrEmpty(codeModule.Object.Content())
+                ? 0
+                : codeModule.Object.Content().GetHashCode());
```

Comment:
gj, I'd have forgotten this.

Reply:
To be honest, I had forgotten it, but a test failed because of it.